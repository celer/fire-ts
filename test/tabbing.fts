# No tabs
<% 
if(true){
	%>
	Hello
	<%
	var colors=["red","green","blue"];
	var numbers=[1,2,3];
	for(var i in colors){ 
		var color=colors[i]; %>
		<%= color%>
		Color <%= color%>
		<% for(var j in numbers){ %>
			<%= numbers[j] %>
			Number <%= numbers[j] %>
			Number <%# numbers[j] %>
			Number & Color <%=color+" "+numbers[j]%>
		<%}%>
	<%	}
}
%>



# One tabs
<% 
if(true){
	%>
	Hello
	<%
	var colors=["red","green","blue"];
	var numbers=[1,2,3];
	for(var i in colors){ 
		var color=colors[i]; %>
			<%= color%>
			Color <%= color%>
		<% for(var j in numbers){ %>
				<%= numbers[j] %>
				Number <%= numbers[j] %>
				Number <%# numbers[j] %>
				Number & Color <%=color+" "+numbers[j]%>
		<%}%>
	<%	}
}
%>
