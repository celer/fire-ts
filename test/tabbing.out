# No tabs
Hello
red
Color red
1
Number 1
Number 1
Number & Color red 1
2
Number 2
Number 2
Number & Color red 2
3
Number 3
Number 3
Number & Color red 3
green
Color green
1
Number 1
Number 1
Number & Color green 1
2
Number 2
Number 2
Number & Color green 2
3
Number 3
Number 3
Number & Color green 3
blue
Color blue
1
Number 1
Number 1
Number & Color blue 1
2
Number 2
Number 2
Number & Color blue 2
3
Number 3
Number 3
Number & Color blue 3



# One tabs
Hello
	red
	Color red
	1
	Number 1
	Number 1
	Number & Color red 1
	2
	Number 2
	Number 2
	Number & Color red 2
	3
	Number 3
	Number 3
	Number & Color red 3
	green
	Color green
	1
	Number 1
	Number 1
	Number & Color green 1
	2
	Number 2
	Number 2
	Number & Color green 2
	3
	Number 3
	Number 3
	Number & Color green 3
	blue
	Color blue
	1
	Number 1
	Number 1
	Number & Color blue 1
	2
	Number 2
	Number 2
	Number & Color blue 2
	3
	Number 3
	Number 3
	Number & Color blue 3
